### ISSUE TITLE
Installation from npm does not work

### ISSUE THREAD

**Issue by JirkaChadima** [windingtree/wt-read-api#206]
1. Run `npm i -g @windingtree/wt-read-api && WT_CONFIG=playground wt-read-api`
2. Watch the startup command fail epically with

```
TypeError: p.a is not a constructor
    at new t (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:2532)
    at Function.value (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:2611)
    at new t (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:37538)
    at Function.value (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:37802)
    at new t (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:40494)
    at Function.value (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/node_modules/@windingtree/wt-js-libs/dist/node/wt-js-libs.js:1:40649)
    at Object.<anonymous> (/home/jirka/.nvm/versions/node/v10.13.0/lib/node_modules/@windingtree/wt-read-api/src/config/playground.js:9:20)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
```

Apparently it's an issue with the compiled wt-js-libs, so it's probably a bug in wt-js-libs. What's weird is that it seems to be working just fine within the docker image.

**Comment by JirkaChadima** [comment]
So, I'm pretty sure this is related to this web3.js release https://github.com/ethereum/web3.js/releases/tag/v1.0.0-beta.38 and the fact that we don't have the older beta pinned in our repositories. Semver FTW. Not.

**Comment by JirkaChadima** [comment]
This can be fixed only after #203 because it needs to pin wt-js-libs to 0.8.1. It would be just too much hassle to do it even for the 0.7.x series.

**Comment by JirkaChadima** [comment]
This is probably the original problem within web3js itself: https://github.com/ethereum/web3.js/issues/2266

**Comment by JirkaChadima** [comment]
Will be hopefully fixed withing #203 by pinning web3 and upgrading wt-js-libs.
